app/views/layouts/_footer.html.erb

```diff
@@ -1,21 +1,17 @@
 <div class="container empty-div">
 </div>
 <div class="text-white">
-  <div class="pl-5 pt-2 container-fluid" style="background: #1c1c1c;">
-    <div class="col-12">
-      <div class="row mt-3 p-3 text-left">
-        <div class="col-lg-3 col-xl-3 mb-3">
-          <div class="row">
-            <div>
+  <div class="p-2 container-fluid" style="background: #1c1c1c;">
```

Comment:
I'd suggest to not use inline styling, rather apply that style inside the CSS file using the id or class of that div.

Reply:
Thanks for your suggestion. I've made the necessary changes and committed the code.